README.md

```diff
@@ -69,6 +69,13 @@ It's based on Rails 4 and Ruby 2.2.2.
 * `bin/ci` - should be used in the CI or locally
 * `bin/server` - to run server locally
 
+## ENV vars
```

Comment:
why not put this comments closer to the described things, right in `.env.example`?